### DISCUSSION TITLE
Powerpoint template to design new layouts

### DISCUSSION THREAD

**Discussion by ahavo** [calibx/msfsdeck#54] [Ideas]
Attached is a powerpoint template that was created early on by Loupedeck as an idea about an MSFS plugin. It's a bit outdated and don't match exactly with the functionality of this plugin, but it might be helpful.

I think the real value of this example might be as a design tool to create new MSFSDeck profile layouts to be used with different planes.

[FS2020 mappings V1.0.pptx](https://github.com/calibx/msfsdeck/files/7979426/FS2020.mappings.V1.0.pptx)

**Comment by calibx** [comment]
Thank you very much for the sharing, it's a great way to complete the plugin. I Will make the missing controls list this weekend.
